api/gen.go

```diff
@@ -45,6 +45,7 @@ import (
     jsmetric "github.com/nats-io/jsm.go/api/jetstream/metric"
 	jsapi "github.com/nats-io/jsm.go/api/jetstream/api"
     scfs "github.com/nats-io/jsm.go/schemas"
+	"github.com/nats-io/nats.go/micro"
```

Comment:
you can roughly ignore these files @aricart this is just plumbing to allow for listing schemas and validating types etc - not consequential.  From you I mainly want a look over the json schemas.